unlock-app/src/components/interface/checkout/main/Confirm/ConfirmCrypto.tsx

```diff
@@ -308,11 +308,6 @@ export function ConfirmCrypto({
                     pricingData.total
                   ).toLocaleString()} ${symbol}`
             }
-            usdPrice={
```

Comment:
No need to show fiat prices (especially as it includes the card fees!) for crypto purchases.